src/manifest/ManifestApiS2S.ts

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-unused-vars */
+
+import { ScopesS2S } from './api/ScopesS2S';
+import { ManifestApiS2SPublicKey } from './ManifestApiS2SPublicKey';
+
+/**
+ * Optional section defining parameters needed for accessing Outreach API.
+ * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/manifest.md#api-optional
+ * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/outreach-api.md
+ * @export
+ * @class ManifestApiS2S
+ */
+export class ManifestApiS2S {
+  /**
+   * The list of scopes will be used for Outreach API authentication
+   * where current Outreach user will be asked to consent for granting
+   * permissions for defined scopes to addon creator.
+   *
+   * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/manifest.md#scopes
+   * @type {ScopesS2S[]}
+   * @memberof ManifestApiS2S
+   */
+  scopes: ScopesS2S[];
+
+  /**
+   *
+   * Outreach API client data used for authentication
+   *
+   * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/manifest.md#client
+   * @type {string}
+   * @memberof ManifestApiS2S
+   */
+  applicationId: string;
```

Comment:
@honeydripper , shall we name this accordingly s2sId?

Reply:
definitely. In the UI we settled for S2S GUID

Reply:
can we go with just `id` maybe? It would we `manifest.apiS2S.id` instead of `manifest.apiS2S.s2sId`.

Reply:
I think `guid` would  be slightly better. @PeSme can you make a definite decision here?

Reply:
both guid and id are fine with me.

Reply:
guid is slightly better

Reply:
Renamed to guid